tests/storage-disks-vm

```diff
@@ -234,8 +175,69 @@ else
   echo 'Skipping NVME test due to missing extension: "disk_io_bus"'
 fi
 
+
+# Check adding a disk to a vm with a long name (max 63) and slash to test possible problems with long socket paths or long qemu device tags
```

Comment:
I would prefer to keep these tests in a place where the project restrictions were still enabled so we are testing that this works with project restrictions (as we are now).